code_testing.ts

```diff
@@ -13,14 +13,13 @@ function test_date_parser() {
 }
 
 function delay(ms: number) {
-  return new Promise(resolve => setTimeout(resolve, ms));
+  return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
 function test_new_function() {
-  let be = new BackendProcessing();
-  be.userSeesFund(1, 1).then(result => {
-    console.log(result);
-  });
+  let object = new Array();
+  object["a"] = "a";
+  object["b"] = "b";
+  console.log(object.keys());
 }
-
```

Comment:
Wasn't this an individual testing file that was accidentally uploaded to the milestones? If so, we should remember to remove it later.